rust/arrow/src/lib.rs

```diff
@@ -124,7 +124,6 @@
 //!
 //! The parquet implementation is on a [separate crate](https://crates.io/crates/parquet)
 
-#![feature(specialization)]
```

Comment:
🎉

Reply:
#arewestableyet? This is great. I think it's just parquet that still uses it after this PR.